buildenv/jenkins/openjdk_tests

```diff
@@ -386,28 +385,23 @@ timestamps{
                         // DynamicAgents and DockerAgents shouldn't contain the same label
                         if (params.CLOUD_PROVIDER != null && params.CLOUD_PROVIDER in dynamicAgents) {
                             if (params.CLOUD_PROVIDER == "EBC") {
-                                boolean ebcNode = false
-                                ebcDemandId = UUID.randomUUID().toString()
-                                def dynamicMachineLabel = 'ci.agent.dynamic.' + ebcDemandId
                                 try {
                                     timeout(time: 120, unit: 'MINUTES') {
                                         // ToDo: pass in platform once we support more platforms. For now, we only support xlinux.
-                                        getEBCNode(ebcDemandId, dynamicMachineLabel)
+                                        def dynamicMachineLabel = 'ci.agent.dynamic.' + UUID.randomUUID().toString()
+                                        getEBCNode(dynamicMachineLabel)
                                         node(dynamicMachineLabel) {
                                             echo "EBC node ${dynamicMachineLabel} is ready"
```

Comment:
Do we need to check for the node to be ready here? It appears that is what param WAIT is doing on the Create job.